docs/aerodromes/classc/Edinburgh.md

```diff
@@ -4,7 +4,74 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name            | Callsign           | Frequency  | Login Identifier |
+| --------------- | ------------------ | ---------- | ---------------- |
+| Edinburgh ADC   | Edinburgh Tower    | 118.300    | ED_TWR           |
+| Edinburgh SMC   | Edinburgh Ground   | 134.100    | ED_GND           |
+
+## Airspace
+<figure markdown>
+![EDN ADC Airspace](img/edncircuits.png){ width="700" }
+</figure>
+
+- **EDN ADC Airspace**: `SFC` to `A015` within the Edinburgh Control Zone (ED CTR).
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+
+The Edinburgh Circuit Area (EDN CIRA) is active at all times when the Edinburgh CTR is active.
+
+Circuit operations occur within the CTR bounded by the Gawler River to the north and the powerline running from the Gawler River (1.5NM south of the Gawler Township) to the south eastern boundary of the ED CTR. Vertical Limits `SFC` to `A020`
+
+If Parafield RWY 03L is active, it can infringe on the circuit deadside. VFR traffic in the ECA is instructed to remain on the west of the RWY centreline.
+
+**Example**:
+- **EDN ADC**: "CIVIL TRANSIT IN PROGRESS, DEAD-SIDE NOT AVAILABLE"
+
+### Circuit Heights
+| Type    | Normal Altitude |
+| ------- | --------------- |
+| Non Jet | A015            |
+| Jet     | A020            |
+
+
+### Circuit Direction
+
+| Runway | Direction |
+| ------ | --------- |
+| 18     | Left      |
+| 36     | Left      |
+
+## Departure Levels
+
+### Departures
+Aircraft can expect to depart to the north, regardless of the duty runway, to conform with Adelaide traffic.
+
+Aircraft will be cleared to an initial level of A040 at clearance delivery unless a lower level is requested. Adelaide Approach will clear aircraft to their planned height following departure.
```

Comment:
Can this be tweaked to match the formatting of other Class D Tower Coordination sections please? Have the departures section, arrivals/overfliers section, and standard assignable levels written within them

Reply:
Not a Class D